added docker usage guide and added explanation for #437 (#503)
ElektraInitiative/PermaplanT: doc/backend/01setup.md

- `AUTH_CLIENT_ID` the client id the frontend should use to log in
- `RUST_LOG` used to set the logging config for [env_logger](https://docs.rs/env_logger/latest/env_logger/)

> The user 'permaplant' doesn't have the required permissions to create the 'postgis' extension. 
> 
> Set it up using the postgres user.
> 
> Ensure that you grant the necessary permissions for the user to use Postgres. One way to do this is by using the following command:

Reviewer: ```suggestion
Ensure that you grant the necessary permissions for the user to use PostgreSQL and that the `postgis` extension is available.
One way to do this is by using the following command:
```
